test/integration/storageversion/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+- roycaihw
```

Comment:
potentially the SIG label should be here:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/apimachinery/OWNERS#L25-L26

not sure if there should be more approvers added here too. ideally, yes.

Reply:
added the label and @caesarxuchao in approvers. Sadly I couldn't find a sig-api-machinery-approvers alias